spec/entities/base_spec.rb

```diff
@@ -51,37 +59,103 @@
   end
 
   describe 'request' do
-    let(:fixture) { File.read("spec/savon/fixtures/project/does_not_exists.xml") }
-    let(:client) { mock('client') }
-    let(:response) { mock('response') }
-    let(:request) { mock('request') }
-    let(:options) { mock('options') }
+    context 'with successful responses' do
+      let(:exists_response) { {'exists' => false} }
+      let(:bases_response) { {'bases' => {'id' => 1}} }
+      let(:prefix_bases_response) { {'bases' => {'id' => 2}} }
+
+      let(:connection) { Faraday.new { |builder| builder.adapter :test do |stubs|
```

Comment:
Is it necessary? There's no way for the code to return the correct answer if it didn't do the requests correctly.